versioned_docs/version-8.7/components/connectors/out-of-the-box-connectors/google-gemini.md

```diff
@@ -220,7 +220,7 @@ Assuming you have created an OAuth client, you can download key files from the G
 from google_auth_oauthlib.flow import InstalledAppFlow
 import pprint
 
-SCOPES = ['https://www.googleapis.com/auth/drive', 'https://www.googleapis.com/auth/documents']
+SCOPES = ['https://www.googleapis.com/auth/cloud-platform', 'https://www.googleapis.com/auth/generative-language.retriever']
```

Comment:
I am not sure if this second link is meant to change -- can you confirm @sbuettner?